src/mpid/ch4/shm/ipc/gpu/gpu_post.h

```diff
@@ -49,7 +38,7 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_GPU_get_mem_attr(const void *vaddr, MPIDI_IPC
 
 #ifdef MPIDI_CH4_SHM_ENABLE_GPU
     attr->ipc_type = MPIDI_IPCI_TYPE__GPU;
-    if (MPIR_CVAR_CH4_GPU_ENABLE) {
+    if (MPIR_ENABLE_GPU) {
```

Comment:
`MPIR_CVAR_ENABLE_GPU`

Reply:
This variable has been deleted in a later commit.

Reply:
I noticed that it was wrong in the commits although it is deleted in a later commit. I fixed all the intermediate commits.